lib/resty/etcd/v3.lua

```diff
@@ -30,7 +32,53 @@ local mt = { __index = _M }
 -- define local refresh function variable
 local refresh_jwt_token
 
-local function _request_uri(self, method, uri, opts, timeout, ignore_auth)
+local function incr(key, shm_name, failure_window)
+    local new_value, err, forcible = ngx_shared[shm_name]:incr(key, 1, 0, failure_window)
```

Comment:
do we need to check `ngx_shared[shm_name]` first?

Reply:
has checked when new etcd client, no need to check here again, right?